Lib/fontTools/feaLib/ast.py

```diff
@@ -2063,9 +2063,10 @@ def __init__(self, name, conditions, location=None):
         Statement.__init__(self, location)
```

Comment:
doesn't `Statement.__init__(self, location)` already store `self.location`?

Reply:
yes, I think that's done in `Element.__init__`, and `Statement` in turn should inherit from Element:

https://github.com/fonttools/fonttools/blob/7b7882355783bc9dd2dd94d6cfcde07c95965ba4/Lib/fontTools/feaLib/ast.py#L163